### ISSUE TITLE
Visual design, art, and creative writing for the civic tech community

### ISSUE THREAD

**Issue by werdnanoslen** [open-austin/project-ideas#111]
### What problem are we trying to solve?

We are not inclusive of more visual-oriented designers, artists, and creative writers. Open Austin itself, too, has good branding but we only really have a logo and basic color scheme right now. All together, we could use more help with a visually appealing brand and more diverse designed media.

### Who will benefit (directly and indirectly) from this project?

Any fans of Open Austin who like swag, as well as anyone who might appreciate fine swag and would thus be more likely to check us out.

### Where can we find any research/data available/articles?

[open-austin.org](https://open-austin.org)

### What help is needed at this time?

Print designers
Visual designers
Graphic artists
Digital artists
Illustrators
Creative writers
Makers

### What are the next steps (validation, research, coding, design)?

We have printed stickers and pins before, and I have access to a screen printing studio. We also have a website, Twitter, Medium, and Facebook accounts that are fairly active and have a good number of followers. We could use more creative visual work to share with the civic tech community!

----

#### Checklist for NEW ideas :baby:
Hey, you're official! You're now part of the growing civic hacking community in Austin. Here's a few things to get started (a couple you've probably already done).

- [x] Create this idea issue
- [x] Flesh out the who, where, and what questions above
- [x] Start the conversation about this new idea on [Slack](https://openaustin-slackin.herokuapp.com/) (this project uses [#comms-team](https://open-austin.slack.com/messages/C069VT41W))
- [ ] Respond with at least one update on this issue within the next month

#### Checklist for ACTIVE projects :fire:
Let's get this project started! When this idea starts taking off, the Projects Core Team will start helping this project's lead(s) out with project management and connecting you to resources you may need. To get there, please complete and check off the following:

- [x] Post an update at least once a month to this issue. Use [BASEDEF](https://github.com/open-austin/cheatsheet-basedef) for ideas, but it's ok even if your update is just "nothing new happened this month" or "we saw a small increase in traffic to our app this month". If there's no activity for two months, that's no problem, life happens. We'll just label this as `backlog` so others know you'll get back to it when you have the time. If nobody hears from you at all in more than two months, we may mark it as `abandoned` so that others can pick up this idea and run with it.
- [x] Take 30 minutes to complete [Open Leadership 101](https://mozilla.teachable.com/p/open-leadership-101) 
- [ ] Create a [GitHub repository](https://github.com/new) and Slack channel for work.
- [ ] Create a [README file](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/write-a-great-project-readme/) in your project repository. This file should help newcomers understand what your project is, why it's important, and kinds of help you're looking for.
- [ ] [Create issues](https://mozilla.github.io/open-leadership-training-series/articles/get-your-project-online/project-set-up-for-collaboration-with-github/#assignment--add-your-first-issue) to describe each task that you plan to do or need help with and how a contributor can get started on that task. You might start and stop a lot, so consider issues as your to-do list.

#### Checklist for FEATURED Projects :tada:
To have your project FEATURED on [Open-Austin.org](https://open-austin.org/), complete the following documentation. In past projects, well-documented featured projects have more contributions than other projects.

* [ ] In your README, link to the [Open Austin Code of Conduct](https://www.open-austin.org/about/#code-of-conduct) or write your own code of conduct.
* [ ] [Create file: LICENSE](http://choosealicense.com/) to give your project an open license, allowing for sharing and remixing.
* [ ] [Create file: CONTRIBUTING.md](https://github.com/acabunoc/mozsprint-repo-template/blob/master/CONTRIBUTING.md) so others know [how they can contribute](https://mozilla.github.io/open-leadership-training-series/articles/building-communities-of-contributors/write-contributor-guidelines/).
* [ ] Create an easily shareable project management artifact, like a [Civic Tech Canvas](https://cityofaustin.github.io/civic-tech-canvas/) or [Open Canvas](https://mozilla.github.io/open-leadership-training-series/articles/opening-your-project/develop-an-open-project-strategy-with-open-canvas/)

Once all of the above is complete,
- [ ] [Create an issue](https://github.com/open-austin/open-austin.github.io/issues/new) on the open-austin.github.io repo with the title `Add [my project] to projects page`. An Open Austin leader will review this issue and post your project :balloon:
- [ ] [Tell the City of Austin](http://austintexas.gov/page/mobile-app-library). If your idea is in a shareable format and can benefit people around the city, go to that site and follow the instructions on the bottom of the page to showcase your work there.

If you get stuck at any point, feel free to reach out to the leadership team on Slack by adding @leadership to your message. We're here to help you make real changes to our city.

**Comment by werdnanoslen** [comment]
First, a call to the community: what are some interesting prompts, themes, and topics we might use as bases to make stuff?
